docs/publish/publish-nested-app-auth-add-in.md

```diff
@@ -0,0 +1,111 @@
+
+The following admin URI example shows how to specify the `User.Read` and `Files.Read` scopes and redirect to a page on your web server named `consentRedirect.html`.
+
+```html
+https://login.microsoftonline.com/organizations/v2.0/adminconsent?client_id=c6c1f32b-5e55-4997-881a-753cc1d563b7&scope=https://graph.microsoft.com/User.Read https://graph.microsoft.com/Files.Read&redirect_uri=https://localhost:3000/consentRedirect.html
+```
+
+> [!IMPORTANT]
+> The redirect page must be added to the list of SPA redirects in your app registration along with the `brk-multihub` redirect or the admin consent URI will fail.
+
+## Get admin consent via the unified manifest
+
+You can also get admin consent as an automatic part of the deployment workflow when your add-in is deployed. To do this you add the `webApplicationInfo` property to your unified manifest. Then the admin deploys the updated manifest, either through central deployment, or from an update through Microsoft AppSource. When the admin deploys the updated manifest they are automatically prompted to consent to the scopes required by the add-in. If they don't consent, the updated add-in will not deploy.
+
+### Add Graph scopes to app registration
+
+When the admin deploys the updated manifest of your add-in, the consent process will read your app registration for which scopes to require the admin for consent. Be sure to add all permissions your add-in requires using the following steps.
+
+1. Sign in to the [Azure Portal](https://portal.azure.com/) and open your app registration.
+
+1. From the left pane, select **API permissions**.
+
+    :::image type="content" source="../images/azure-portal-api-permissions.png" alt-text="The API permissions pane.":::
+
+    The **API permissions** pane opens.
+
+1. Select **Add a permission**.
+
+    :::image type="content" source="../images/azure-portal-add-a-permission.png" alt-text="Adding a permission on the API permissions pane.":::
+
+    The **Request API permissions** pane opens.
+
+1. Select **Microsoft Graph**.
+
+    :::image type="content" source="../images/azure-portal-request-api-permissions-graph.png" alt-text="The Request API permissions pane with Microsoft Graph button.":::
+
+1. Select **Delegated permissions**.
+
+    :::image type="content" source="../images/azure-portal-request-api-permissions-delegated.png" alt-text="The Request API permissions pane with delegated permissions button.":::
+
+1. In the **Select permissions** search box, search for the permissions your add-in needs. For example, for an Outlook add-in, you might use `profile`, `openid`, `Files.ReadWrite`, and `Mail.Read`.
+
+1. Select the checkbox for each permission as it appears. Note that the permissions will not remain visible in the list as you select each one. After selecting the permissions that your add-in needs, select **Add permissions**.
```

Comment:
What happens if they wish to deselect a permission they've chosen if it's no longer visible? Just cancel and start over?